packages/lesswrong/components/recentDiscussion/RecentDiscussionTagRevisionItem.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { forumTypeSetting } from "../../lib/instanceSettings"
+import { Components, registerComponent } from "../../lib/vulcan-lib"
+
+const isEAForum = forumTypeSetting.get() === 'EAForum'
+// Pablo, Leo
+const megaTagUsers = ['BkbwT5TzSj4aRxJMN', 'pkJTc4xXhsCbNqkZM']
+const onlyStyleEditors = ['pkJTc4xXhsCbNqkZM']
+
+/**
+ * This component's only job is to reduce the amount of room the EA frontpage
+ * gives to the most particularly active tag users doing routine cleanup
+ *
+ * Otherwise it's just a wrapper around TagRevisionItem
```

Comment:
I have an architectural comment! I think excluding these authors should be a decision that's made by the list of revisions, not by each individual revision. So I claim this component should have all of the revisions as an input, and additionally the sorting (and maybe also pagination) logic should live here.

Reply:
Um. I think you're confused about how the RecentDiscussion section works. Perhaps you're confusing it with the allPosts page tag edit section? There's no component that gets all the revisions that will appear in Recent Discussion, except perhaps `RecentDiscussionFeed.tsx`, but I think if you look at that component you'll agree it's the wrong place to filter. The way I'd like to filter it is in `resolvers/recentDiscussionFeed.ts`, but I don't fancy the mongo query I'd have to do.